webpack.config.js

```diff
@@ -30,7 +30,9 @@ module.exports = {
           {
             loader: "css-loader",
             options: {
-              modules: true
+              modules: {
+                localIdentName: "[name]__[local]___[hash:base64:5]"
```

Comment:
To make this even better, we could add `esm-home__` to the beginning of the string so that (in the very rare likelihood of) collisions between the base64 hashes in different modules don't ever cause a problem.